test/unit_tests/experiments/test_scoring.py

```diff
@@ -0,0 +1,106 @@
+# Authors: Maciej Sliwowski
+#          Robin Tibor Schirrmeister
+#
+# License: BSD-3
+
+
+import numpy as np
+import sklearn.datasets
+import torch
+from sklearn.metrics import f1_score, accuracy_score
+from skorch.callbacks import Callback
+from torch import optim
+from torch.utils.data import Dataset
+
+from braindecode.experiments.classifier import BraindecodeClassifier
```

Comment:
mayeb BraindecodeClassifier -> EEGClassifier ?
in a braindecode package calling an object iwht the package name seems redundant